go.mod

```diff
@@ -0,0 +1,22 @@
+module github.com/chihaya/chihaya
+
+require (
+	github.com/SermoDigital/jose v0.0.0-20180104203859-803625baeddc
+	github.com/alicebob/gopher-json v0.0.0-20180125190556-5a6b3ba71ee6 // indirect
+	github.com/alicebob/miniredis v2.4.6+incompatible
+	github.com/anacrolix/torrent v1.0.0
```

Comment:
I would have expected anacrolix/missinggo and anacrolix/dht to appear in your indirect list. Unless it's because anacrolix/torrent has a go mod file?

Reply:
That very well could be. I just blew everything away and regenerated a new go.mod and the only difference is one thing that I hand edited with the jose library, which is because they don't use semver for their git tags.

Reply:
Meh, it works 👍

Reply:
> Unless it's because anacrolix/torrent has a go mod file?

That's exactly why :) See [here](https://github.com/golang/go/wiki/Modules#why-does-go-mod-tidy-record-indirect-and-test-dependencies-in-my-gomod), 4th paragraph:

> If a dependency of your module does not itself have a go.mod (e.g., because the dependency has not yet opted in to modules itself), or if its go.mod file is missing one or more of its dependencies (e.g., because the module author did not run go mod tidy), then the missing transitive dependencies will be added to your module's requirements, along with an // indirect comment to indicate that the dependency is not from a direct import within your module.